packages/admin/src/index.ts

```diff
@@ -126,3 +126,10 @@ export { ErrorDialogProvider } from "./error/errordialog/ErrorDialogProvider";
 export { useErrorDialog } from "./error/errordialog/useErrorDialog";
 export { ErrorScope, LocalErrorScopeApolloContext, errorScopeForOperationContext } from "./error/errordialog/ErrorScope";
 export { ErrorDialogApolloLinkOptions, createErrorDialogApolloLink } from "./error/errordialog/createErrorDialogApolloLink";
+export * from "./common/buttons/split/SplitButton";
+export { CometAdminSplitButtonThemeProps } from "./common/buttons/split/SplitButton.styles";
+export * from "./common/buttons/savesplit/save/SaveButton";
+export { CometAdminSaveButtonThemeProps, CometAdminSaveButtonClassKeys } from "./common/buttons/savesplit/save/SaveButton.styles";
+export * from "./common/buttons/savesplit/SaveSplitButton";
+export * from "./common/buttons/savesplit/SaveSplitButtonContext";
+export * from "./common/buttons/savesplit/useSaveSplitButtonContext";
```

Comment:
I'd prefer named exports. With this approach, we can't tell what the public API is